.github/workflows/ci.yml

```diff
@@ -30,10 +31,24 @@ jobs:
         with:
           path: deps
           key: ${{ matrix.os }}-${{ matrix.otp }}-${{ matrix.elixir }}-mix-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+      - name: Retrieve libtorch cache
+        if: ${{ matrix.working_directory == 'torchx' }}
+        uses: actions/cache@v1
+        id: libtorch-cache
+        with:
+          path: libtorch
+          key: libtorch.zip
+      - name: Download and install libtorch
+        if: ${{ matrix.working_directory == 'torchx' && steps.libtorch-cache.outputs.cache-hit != 'true' }}
+        run: |
+          if [ ! -d libtorch ]; then
```

Comment:
If the cache doesn't hit we don't expect the dir there right? And if for some reason it's there it's probably not what we want :)

Reply:
yeah :)